arquillian-ape-sql/core-dbunit/src/main/java/org/arquillian/ape/rdbms/dbunit/DbUnitPopulatorService.java

```diff
@@ -109,6 +109,11 @@ public void clean(List<String> resources) {
     private Optional<IDatabaseConnection> lookupDataSourceConnection(String dataSourceName) {
 
         try {
+
+            if (contextInstance == null) {
+                // In case of running outside Arquillian Runner
```

Comment:
I agree but for this specific use case I think we can leave as is for now. In long term we could inject some `stubbed` classes.